themes/digital.gov/src/scss/new/_resources.scss

```diff
@@ -1,7 +1,20 @@
 @use "uswds-core" as *;
 
+
+// Overide uswds styles for integrating in-page navigation on right column
+
+.grid-container.grid-container-desktop:has(div.usa-in-page-nav-container) {
+  // border-top: 1px solid #dfe1e2;
+}
+
+// nav.usa-nav {
+//   border-bottom: 1px solid #dfe1e2;
+// }
 // Styles for the main resources page
 
+.resources__container {
+  border-top: 1px solid #dfe1e2;
```

Comment:
Where is this class used?

---

Consider using a USWDS theme color token like `base-lighter` (`#dfe1e2`).